src/wp-includes/user.php

```diff
@@ -3172,7 +3172,7 @@ function retrieve_password( $user_login = '' ) {
 	$user_data = false;
 
 	// Use the passed $user_login if available, otherwise use $_POST['user_login'].
-	if ( ! $user_login && ! empty( $_POST['user_login'] ) ) {
+	if ( ! $user_login && ! empty( $_POST['user_login'] ) && is_string( $_POST['user_login'] ) ) {
```

Comment:
How i reproduce the issue?

Reply:
@mukeshpanchal27 Please check the ticket description.

![image](https://github.com/user-attachments/assets/10bb8eb9-8445-4e56-a685-6056a160ba3c)